### PR TITLE
Ticket4581 front detector beam stop

### PR DESCRIPTION
**Pull request by JamesKingWork**
### Description of work

Add records to inhibit movement of the front beam stop axis and motor when the sp or rbv of the front detector rotation is greater than 0.

### To test

https://github.com/ISISComputingGroup/IBEX/issues/4581

### Acceptance criteria

1. If the detector rotation is greater than 0 degrees, motion of the front beam stop must be inhibited.

---

#### Code Review

- [ ] A copy of the manual has been placed on the shared drive
- [ ] Pertitent information has been stored in the [wiki](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Specific-Device-IOC)
- [ ] Does the IOC conform to IBEX standards?
    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)
- [ ] If an OPI has been modified, does it conform to the [style guidelines](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/OPI-Creation)?
- [ ] Do the IOC and support module conform to the new [build guidelines](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Reducing-Build-Dependencies)
- [ ] Have the changes been recorded appropriately in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?
- [ ] Is the device's flow control setting correct? [For most devices flow control should be OFF](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Flow-control).

### Functional Tests

- IOC responds correctly in:
    - [ ] Devsim mode
    - [ ] Recsim mode
    - [ ] Real device, if available
- [ ] Supplementary IOCs (`..._0n` where `n>1`) run correctly
- [ ] Log files do not report undefined macros (serach for `macLib: macro` to find instances of `macLib: macro [macro name] is undefined...`

### Final steps

- [ ] Update the IOC submodule in the main EPICS repo. See [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    field(DESC, "Inhibit fbs movement if det rot > 0")
+    field(INPA, "$(P)MOT:FRONTDETROT CP")
+    field(INPB, "$(P)MOT:FRONTDETROT:SP CP")
+    field(CALC, "A > 0 || B > 0")
+    field(OUT, "$(FULL_PREFIX)_INHIBIT_FRONTBEAMSTOP PP MS")
+}
+
+record(dfanout, "$(FULL_PREFIX)_INHIBIT_FRONTBEAMSTOP") {
+    field(DESC, "Stop and inhibit fbs movement")
+    field(DOL, "$(FULL_PREFIX)INHIBIT_FRONTBEAMSTOP PP MS")
```
Comment (FreddieAkeroyd) [inline FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template:11]: Why is this input PP ?

File: FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    field(OUTA, "$(P)MOT:FRONTBEAMSTOP:MTR.STOP")
```
Comment (FreddieAkeroyd) [inline FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template:12]: Does STOP need to be PP ?

File: FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template (hunk 1 above)
Comment (JamesKingWork) [reply FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template]: I think I meant to put this PP against stop, swapped now

**Comment by JamesKingWork** [comment]
Closing as moved to motorExtensions

**Pull request closed without merging**